deploy/olm-catalog/service-telemetry-operator/manifests/service-telemetry-operator.clusterserviceversion.yaml

```diff
@@ -64,7 +64,7 @@ metadata:
               "metrics": {
                 "prometheus": {
                   "enabled": true,
-                  "scrapeInterval": "10s",
+                  "scrapeInterval": "30s",
```

Comment:
??

Reply:
This _should_ go away if the branch is updated. I missed this originally at one point, and had recently fixed it before importing to stable-1.5.